website/docs/secrets/bootstraping-secrets.mdx

```diff
@@ -5,17 +5,218 @@ hide_title: true
 
 ## Using `SecretSync` Custom Resource
-// Description on how SecretSync works and what it provides and its limitations, and maybe an example CR yaml
+
+`SecretSync` is a [Kubernetes Customer Resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+that provides semantics to sync [Kuberentes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) from management cluster to leaf clusters.
+
+An example could be seen below:
+
+```yaml
+apiVersion: capi.weave.works/v1alpha1
+kind: SecretSync
+metadata:
+  name: my-dev-secret-syncer
+  namespace: default
+spec:
+  clusterSelector:
+    matchLabels:
+      environment: dev
+  secretRef:
+    name: my-dev-secret
+  targetNamespace: my-namespace
+```
+Where you could find two main configuration sections:
+
+1) Select the secret to sync:
+
+```yaml
+  secretRef:
+    name: my-dev-secret
+```
+
+2) Select the [GitopsClusters](https://docs.gitops.weave.works/docs/cluster-management/managing-existing-clusters/)
+,that the secret will be synced to, via labels:
+
+```yaml
+  clusterSelector:
+    matchLabels:
+      environment: dev
+```
+
+More info about the CRD spec [here](./spec/v1alpha1/secretSync.mdx)
+
 ### Working with SecretSync
-// Step by step guide to how to set it up to acheive bootstrapping a secret from management cluster to a new cluster called 'dev'
 
+#### Before start
+
+1. You are using [Weave Gitops Enterprise version > v0.19.0](https://docs.gitops.weave.works/docs/releases/#v0190)
+2. You are running cluster-bootstrap-controller > v0.5.0
+2. You have a secret that you want to sync from the management cluster.
+3. You have a set of GitopsClusters that you want to sync to.
+
+#### Syncing secrets via SecretSync
+
+1. Create SecretSync manifests using the secret for example
+
+```yaml
+apiVersion: capi.weave.works/v1alpha1
+kind: SecretSync
+metadata:
+  name: my-dev-secret-syncer
+  namespace: default
+spec:
+  clusterSelector:
+    matchLabels:
+      environment: dev
+  secretRef:
+    name: my-dev-secret
+  targetNamespace: my-namespace
+```
+
+2. Check-in to your configuration repo within your management cluster
+
+3. Create a PR, review and merge
+
+4. See the progress
+
+```bash
+kubectl get secretsyncs.capi.weave.works my-dev-secret-syncer -o yaml
+
+apiVersion: capi.weave.works/v1alpha1
+kind: SecretSync
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"capi.weave.works/v1alpha1","kind":"SecretSync","metadata":{"annotations":{},"name":"my-dev-secret-syncer","namespace":"default"},"spec":{"clusterSelector":{"matchLabels":{"environment":"dev"}},"secretRef":{"name":"my-dev-secret"},"targetNamespace":"my-namespace"}}
+  creationTimestamp: "2023-03-28T16:44:37Z"
+  generation: 1
+  name: my-dev-secret-syncer
+  namespace: default
+  resourceVersion: "4980"
+  uid: f6670578-4403-46f8-a9b5-de8c80559c2b
+spec:
+  clusterSelector:
```

Comment:
to review with @ahsayde and bootstrap controller v0.5.0

I cannot set the resource being reconciled